internal/conv/conv.go

```diff
@@ -17,7 +17,6 @@ package conv
 
 import (
 	"io"
```

Comment:
Is your fix the mistake?
I commented using` internal/io` instead of `io`.

Reply:
Sorry I made mistake. but I couldn't use `internal/io` as it is cycle import in `internal/io/io.go`.

```
package github.com/vdaas/vald/internal/conv
	imports github.com/vdaas/vald/internal/io
	imports github.com/vdaas/vald/internal/conv: import cycle not allowed
```